product_docs/docs/transporter/ba_preview/config_reader.mdx

```diff
@@ -0,0 +1,54 @@
+---
+title: "Configuring"
+---
+
+## Config a source reader
+
+### Get credentials
+
+- Create and download a migration credentials from the postgres AI platform Migrate -> Credentials section
+- un-zip the credential and copy the whole folder to the host where the reader is installed
+```shell
+```
+
+### Configure the reader
+
+Set the following environment variables in /opt/cdcreader/run-cdcreader.sh with the right values:
+
+```shell
+### set the following environment variables:
+
+##########################################
+# Transporter Cloud Configuration        #
+##########################################
+
+# This ID is used to identify the cdcreader.
+#export DBZ_ID="my_reader_id"
+#export RW_SERVICE_HOST=https://transporter-rw-service.biganimal.com
+#export TLS_PRIVATE_KEY_PATH=$MY_CREDENTIALS_PATH/client-key.pem
+#export TLS_CERTIFICATE_PATH=$MY_CREDENTIALS_PATH/client-cert.pem
+#export TLS_CA_PATH=$MY_CREDENTIALS_PATH/int.crt
+#export APICURIOREQUEST_CLIENT_KEYSTORE_LOCATION=$MY_CREDENTIALS_PATH/client.keystore.p12
+#export APICURIOREQUEST_TRUSTSTORE_LOCATION=$MY_CREDENTIALS_PATH/int.truststore.p12
+#export KAFKASECURITY_CLIENT_KEYSTORE_LOCATION=$MY_CREDENTIALS_PATH/client.keystore.p12
+#export KAFKASECURITY_TRUSTSTORE_LOCATION=$MY_CREDENTIALS_PATH/int.truststore.p12
+
+##########################################
+# Transporter Source DB Configuration    #
+##########################################
+
+# A sample configuration to create a single database connection:
```

Comment:
The file I git during testing had an example for postgres, and another for oracle, which I found useful. I think we could update it here to. 

Also, in my test, the file contained a typo: 
```
# You can increase the index to config more database for the reader
#export DBZ_DATABASES_1__TYPE=ORALCE
```
It should be Oracle instead or Oralco.

Reply:
fixed